online-shopping-website/src/Components/ProductGrid.js

```diff
@@ -3,47 +3,55 @@ import ButtonBase from '@mui/material/ButtonBase';
 import Grid from '@mui/material/Grid';
 import React from 'react';
 import Stack from '@mui/material/Stack';
+import {ProductDetails} from './ProductDetails';
+import {BrowserRouter, Outlet, Link, Routes, Route} from "react-router-dom";
 
 const ProductPreview = (props) => {
-  return (
-    <ButtonBase sx={{ width: '100%', textAlign: 'left', margin: 0, padding: 0, borderRadius: '15px'}} >
-      <Card className="ProductContainer">
-        <Stack sx={{ height: '100%' }} direction="column" justifyContent="space-between">
-          {/* Image */}
-          <Stack direction="row" justifyContent="center" sx={{ textAlign: 'center' }}>
-            <img className="ProductImage" src={props.product.image} alt={props.product.name} />
-          </Stack>
+    return (
+        // Navigates to ProductDetails page on click
+        <Link to={{
```

Comment:
You may need to find a way to reset the current scroll position when entering a product view more page. After selecting a product, we can no longer see the top of the website (where the navbar would be) because the scroll sticks from the previous page. Resetting it to the top of the page should fix this!